### DISCUSSION TITLE
Modify someone else's kata

### DISCUSSION THREAD

**Discussion by metasean** [codewars/codewars.com#2415] [Q&A] [answered]
I just completed a Kata that could have a different solution enabled _if_ the Kata itself had Node v12 enabled. Unfortunately, the Kata's tests need to be modified to enable Node v12 support (per [this archived response on a related codewars-runner-cli question](https://github.com/codewars/codewars-runner-cli/issues/816#issuecomment-663324423)).

I'm willing to take a stab at updating the tests themselves, but when I tried making the updates via the "translation" route, neither "JavaScript" nor "Node" were options in the "Choose Language...".

So, how do I (a) edit the existing Kata or (b) add a translation for _a different version_ of a language for which the Kata already has a translation?

**Comment by hobovsky** [comment, marked as answer]
There's a known problem that translations which were created directly with kata editor (including initial kata language) cannot be modified by forking. You can fork only translations created via Translations panel.

JS language was probably initial language for the kata you are trying to edit, and thats why it can be fixed only with the kata editor. Depending on your privileges, you might or might not be able to do this. This is already recognised as a problem and a subject to future improvement.

That being said, usually there's no need to keep separate solution for Node 12. Its fine to update the kata to make it Node 12 compatible and do not worry too much about backwards compatibility of tests with prevoius versions.

**Reply by metasean** [reply]
I did finally locate the [criteria to edit](https://docs.codewars.com/gamification/privileges/#required-honor-points) an existing Kata.

The only reason I was trying to translate it was to enable support for Node 12, but based on your comment and the fact that I don't meet any of the criteria to edit, I guess the Kata will simply be limited to Node 8 and 10 until someone else fixes it. 😞
